puppet/modules/web/manifests/vhost/stagingdeb.pp

```diff
@@ -18,8 +18,11 @@
   if $setup_receiver {
     secure_ssh::rsync::receiver_setup { $user:
       user           => $user,
+      homedir        => $home,
       foreman_search => 'host.hostgroup = Debian and (name = external_ip4 or name = external_ip6)',
       script_content => template('freight/rsync.erb'),
     }
+
+    File <| title == $home |> { mode => '0755' }
```

Comment:
technically, this could be 0750 if we add the `freight` user into the `freightstage` group, but I was lazy.

Reply:
which I did now